data/processed/hesa/area_university_site.yaml

```diff
@@ -0,0 +1,23 @@
+api: fetch
+api_documentation: https://www.hesa.ac.uk/data-and-analysis/estates/table-1
+description: Area of university sites for universities in the NUTS2 region
+endpoint_static: https://www.hesa.ac.uk/data-and-analysis/estates/data.csv
+framework_group: public_rnd_capability
+format: .3f
```

Comment:
This should be move under `value`.

Also we need to add the line:
```
order: [year, nuts_id, nuts_year_spec, value.id]
```
see #131 and #130.

Reply:
Done